text/0000-core-builder.md

```diff
@@ -0,0 +1,46 @@
+
+Users will be able to use the new builder to build and run their applications using any of the language family buildpacks included. Instead of requireing a decision for an optimised builder, they can "just" start and later decide on an optimised `run` image.
+
+## Motivation
+
+The Paketo project will be easier to use for users - they can start without being asked to decide on optimisation.
+The stacks and builders maintainer team will have less work to do - they will only need to maintain a single stack and builder instead of three.
+
+## Detailed Explanation
+
+We would replace the current `base`, `tiny` and `static` stacks with a single stack that includes all packages required by the buildpacks. Similarly, we would replace the current `base`, `tiny` and `static` builders with a single builder.
+
+## Rationale and Alternatives
+
+We could leave everything as it is. We would stick to maintaining four stacks repositories and eight builder repositories and creating new ones for every LTS we adopt. But that would increase the burden on the maintainers.
+
+## Implementation
+
+I would propose to create the new stack and builder as part of moving to Ubuntu Noble Numbat. I.e. 
+- we should create a repository `paketo-buildpacks/stack-ubuntu-noble`
+- we should take the `build` image content from the `paketo-buildpacks/noble-base-stack` repository and apply it to `paketo-buildpacks/stack-ubuntu-noble`
+- we should take the `run` image content from the `paketo-buildpacks/noble-base-stack` repository and apply it to `paketo-buildpacks/stack-ubuntu-noble`
+- we should take the `run` image content from the `paketo-buildpacks/noble-tiny-stack` repository and apply it to `paketo-buildpacks/stack-ubuntu-noble` as additional run image
+- we should take the `run` image content from the `paketo-buildpacks/noble-static-stack` repository and apply it to `paketo-buildpacks/stack-ubuntu-noble` as additional run image
+- we should create a repository `paketo-buildpacks/builder-ubuntu-noble` and another`paketo-buildpacks/builder-ubuntu-noble-buildpackless`
+    - we should use the above `build` image as a base layer for the `builder` image
+    - we should use the above `run` image as the default `run` image for the `builder.toml`
+    - we should use the above `run-static` and `run-tiny` images as additional `run` images for the `builder.toml`
+
+I would propose to delete the `paketo-buildpacks/noble-base-stack`, `paketo-buildpacks/noble-tiny-stack` and `paketo-buildpacks/noble-static-stack` repositories.
+I would propose to delete the `paketo-buildpacks/builder-noble-buildpackless-static`, `paketo-buildpacks/builder-noble-buildpackless-tiny` and `paketo-buildpacks/builder-noble-buildpackless-base` repositories.
+
+## Prior Art
+
+None.
+
+## Unresolved Questions and Bikeshedding
+
```

Comment:
Q: How does this impact language-specific builders?

I feel like the two are orthogonal, so this effort won't really change what is/can be done with a language-specific builder. It might be worth clarifying the intent though.

Reply:
Actually, as @pacostas points out below, there could be language specific run images instead of (or in addition to) language specific builders.

Reply:
Yes, but for each language we need to have the corresponding lanuage-builder, which is not that bad, but we will end up having as many builders as the languages are. Also, I think buildpacks are supposed to be a little bit more abstract when we build an app e.g. using builder-ubuntu instead of builder-nodejs-ubuntu.

Reply:
@pacostas I guess what @dmikusa is hinting at is the relation to https://github.com/paketo-buildpacks/rfcs/blob/main/text/0055-create-language-family-builders.md

Reply:
@dmikusa If you think the notion of language-specific builder and/or run images should be included in the RFC, would you mind adding a concrete text change suggestion. Thanks.

Reply:
```suggestion
Q: How does this impact the [language-specific builders RFC](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0055-create-language-family-builders.md)?
A: It does not impact language-specific builders and Paketo buildpacks language families can continue to ship language-specific builders as needed.

```